src/modes/overview/vegaSpec.js

```diff
@@ -121,10 +126,49 @@ export function createSpec(sensor, selections, dateRange, valuePatch) {
     transform: [
       ...(sensor.hasStdErr ? (isPercentage ? stdErrTransformPercent : stdErrTransform) : []),
       {
-        calculate: 'datum.value == null ? null : datum.value / 100',
         as: 'pValue',
+        calculate: 'datum.value == null ? null : datum.value / 100',
+      },
+      {
+        as: 'clipped',
+        calculate: !clipping + '|| datum.' + yField + ' == null ? false : datum.' + yField + ' > ' + yMaxScaled,
+      },
+      {
+        as: 'notClipped',
+        calculate: 'datum.clipped == null ? false : !datum.clipped',
+      },
+      {
+        as: 'clippedData',
+        calculate: 'datum.clipped ? datum.' + yField + ' : null',
+      },
+      {
+        window: [
+          {
+            as: 'previousNotClipped',
```

Comment:
I'm wondering whether it should be the first value that is clipped (first_value) or the first before it is clipped (lag).

Reply:
The clipped region should start and end with clipped values so that the region is completely flat.  That's why we want to determine whether the current value is clipped and the previous value is not clipped.

To compute previousNotClipped we look at the previous value (lag of 1) to check if it is not clipped, so that this will be the start of the clipped region, but only if the current value _is_ clipped.  I would collapse logic into fewer steps if I could figure out how.

For the end of the clipped region, the logic is similar, but the default lead (1) seemed to be 1 too far, hence the param: 0.

This is all fairly confusing.  I'll add a comment about it.